kolena/workflow/test_run.py

```diff
@@ -126,11 +126,15 @@ def __init__(
         self.model = model
         self.test_suite = test_suite
         self.evaluator = evaluator
-        self.configurations = self.evaluator.configurations if is_evaluator_class else configurations
+        self.configurations = self.evaluator.configurations if is_evaluator_class else configurations  # type: ignore
         self.reset = reset
 
         evaluator_display_name = (
-            None if evaluator is None else evaluator.display_name() if is_evaluator_class else evaluator.__name__
+            None
+            if evaluator is None
+            else evaluator.display_name()
+            if is_evaluator_class
+            else evaluator.__name__  # type: ignore
```

Comment:
I think this actually caught a potentially valid issue, not all callable has `__name__`. Proposed resolution

```
def get_evaluator_display_name(evaluator: Optional[Union[Evaluator, BasicEvaluatorFunction]]) -> Optional[str]:
    if not evaluator:
        return None
    if isinstance(evaluator, Evaluator):
        return evaluator.display_name()
    else:
        return getattr(evaluator, "__name__", repr(evaluator))
```
and use
```
evaluator_display_name = get_evaluator_display_name(evaluator)
```

Reply:
Updated to remove type ignore